### ISSUE TITLE
linux: don't surpport chinese

### ISSUE THREAD

**Issue by ethan-phu** [zed-industries/zed#9310]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

![image](https://github.com/zed-industries/zed/assets/12579229/0fb2818e-b714-41a1-bfa1-f0ab6d0df5f5)

### Environment

deeppin v23

**Comment by jansol** [comment]
@ethan-phu Do you have Noto Sans fonts installed? Looks like Zed tries to specifically fall back to those if the main font does not contain some glyphs.

**Comment by aohanhongzhi** [comment]
me too.
![image](https://github.com/zed-industries/zed/assets/37319319/28a17da8-dbfc-4699-8e76-f034d18ebf8c)


but I run the gpui by Zed.

```
use gpui::*;

struct HelloWorld {
    text: SharedString,
}

impl Render for HelloWorld {
    fn render(&mut self, _cx: &mut ViewContext<Self>) -> impl IntoElement {
        div()
            .flex()
            .bg(rgb(0x2e7d32))
            .size_full()
            .justify_center()
            .items_center()
            .text_xl()
            .text_color(rgb(0xffffff))
            .child(format!("中文正常 Hello, {}!", &self.text))
    }
}

fn main() {
    App::new().run(|cx: &mut AppContext| {
        cx.open_window(WindowOptions::default(), |cx| {
            cx.new_view(|_cx| HelloWorld {
                text: "World".into(),
            })
        });
    });
}
```

it works.

![image](https://github.com/zed-industries/zed/assets/37319319/080bef0a-b2f8-4cd9-a1d2-8cc0bd1ea053)

**Comment by aohanhongzhi** [comment]
I solve the problem , I config the font.

https://zed.dev/docs/configuring-zed

```json
{
  "ui_font_size": 16,
  "buffer_font_size": 16,
  "buffer_font_family": "DejaVu Sans Mono",
  "terminal": {
    "font_family": "DejaVu Sans Mono",
    "blinking": "off"
  }
}
```

```shell
fc-list | grep noto
```
![image](https://github.com/zed-industries/zed/assets/37319319/c120a9e0-c6f0-471b-946a-1c73f34ed0c8)


editor and terminal is ok

![image](https://github.com/zed-industries/zed/assets/37319319/b233552e-47fd-4abe-a212-93624320c716)

**Comment by jansol** [comment]
@aohanhongzhi you need to use the `"ui_font_family"` setting to change the font used in the project panel.

**Comment by aohanhongzhi** [comment]
@jansol  you are right

![image](https://github.com/zed-industries/zed/assets/37319319/c31ff2cb-d741-4dc0-9a6c-3b91549d883b)

**Comment by jansol** [comment]
@nt8er there is "a" fallback logic, but the list of fonts in that is currently hardcoded... Definitely looking forward to proper fontconfig support for both that and for font aliases so I can just set the UI font to "sans-serif" and editor font to "monospace" and have it use the same fonts as all other well-behaved apps.

**Comment by ConradIrwin** [comment]
This specific issue is "fixed" by #13568. It turns out Zed Mono was a font with stretch 7, but almost all fonts have stretch 5, so cosmic texts fallback stack was being ignored.

We should track migrating to fontconfig somewhere (there's an open question of whether we need to switch away from cosmic_text to do that, or if we can just upstream some changes).
